README.md

```diff
@@ -10,7 +10,56 @@ See the [release FAQ](https://go.armory.io/release-faq).
+When Astrolabe opens a PR against this repo we treat it as a Release Candidate (RC), so we run an End to End test suit to validate the new service version against the current stack.
+
+### Armory End-to-end Pipelines
+
+The `Deploy Test Staging` job is responsible to deploy the new service version into SaaS staging, the SaaS staging environment has a Spinnaker instance with the service versions described in the [stack.yml](https://github.com/armory-io/armory-stack/blob/master/stack.yml).
+The configuration for the SaaS staging env is located in the [spinnaker-saas](https://github.com/armory-io/spinnaker-saas/tree/master/hosted-core-infrastructure/spinnaker-staging) repo.
+
+We use a Spinnaker [pipeline](https://spinnaker-staging.cloud.armory.io/#/applications/stagingcd/executions?pipeline=Deploy%20Spinnaker%20into%20SaaS%20Staging) in staging to deploy the new service version.
+If the new service version is deployed successfully, we then [run our pipeline-based end-to-end tests](https://spinnaker-staging.cloud.armory.io/#/applications/armoryhellodeploy/executions?pipeline=Integration-Test-Runner) against the updated Spinnaker.
+
+#### Images & Commits
+
+In the LTS release, every commit in the Armory extension projects generates a new Docker image, Astrolabe opens a PR with the new Docker image & the commit that the image belongs to.
+You can find the image tag & the commit in the body of the [PR](https://github.com/armory-io/armory-stack/pull/343#issue-685533758). We can find all the images from the LTS release in [Artifactory](https://armory.jfrog.io/ui/repos/tree/General/docker-all)
+
+The Armory extension projects uses an OSS Service as base, currently the extension projects uses a LTS version that represents a commit in the OSS. A base artifact can be found [here](https://github.com/armory-io/orca-armory/blob/a08ea272f752ebcce01e59e790abbe0381b10d00/gradle.properties#L2) in the extension project. The base service version is maintained by Astrolabe.
+
+The base artifacts points to a commit in the OSS projects but those artifact are [jars](https://armory.jfrog.io/ui/repos/tree/General/gradle-dev-local%2Fio%2Fspinnaker) or npm packages in Artifactory. The [build ID](https://armory.jfrog.io/ui/repos/tree/Builds/gradle-dev-local%2Fio%2Fspinnaker%2Forca%2Forca-core%2F2021.07.07.21.31.05.master%2Forca-core-2021.07.07.21.31.05.master.jar) of the artifact is the commit sha from the OSS project.
+
+#### Deploy Pipeline
+
+We use Spinnaker to upgrade itself, the `Deploy RC to Staging` workflow enqueue PR checks, so we deploy & test a new service version at a time
```

Comment:
"We use Spinnaker to upgrade itself using the `Deploy RC to Staging` workflow enqueue PR checks so we deploy & test one new service version at a time."